source/community/release_notes/teak/dev_op_release_notes.rst

```diff
@@ -306,6 +381,18 @@ Deprecations & Removals
   <https://github.com/openedx/brand-openedx/issues/23>`_, and follow the
   :ref:`Ulmo Design Tokens` page for more detail. Operators will be able to try
   out Design Tokens using the Teak Design Tokens branches (link TBD)
+- `[DEPR]: block_structure.storage_backing_for_cache in edx-platform <https://github.com/openedx/public-engineering/issues/32>`_ This is a simplification to how course content is cached. It should be invisible to all end users.
```

Comment:
@feanil could you or someone from DEPR review the DEPR tickets in the release notes and let us know if there are others to be added? I find it hard to believe there's only been 3 since late October.